src/sections/Company/Layer5-statement/statement.style.js

```diff
@@ -37,6 +37,9 @@ const BannerSectionWrapper = styled.section`
         color: ${(props) => props.theme.white};
         margin-bottom: 1rem;
     }
+    span.span-text{
+        color: white;
```

Comment:
```suggestion
        color: ${(props) => props.theme.white};
```
what do you think about this @Mohith234 ?

Reply:
@thoughtlessmind The background image color doesn't change when the theme changes. I think the text color is also independent of the theme.

Do I still need to change?

Reply:
Hey @Mohith234
sorry for the delay. I've less context on that, maybe someone with more context on that can clarify that
thanks

Reply:
@Mohith234 using `color: ${(props) => props.theme.white};` would be fine.